src/brs/at/AtController.java

```diff
@@ -517,35 +519,51 @@ private static int getCostOfOneAT() {
 
   //platform based implementations
   //platform based
```

Comment:
```suggestion
```

Reply:
weird old comments... let's remove them - Boy Scout Principle: _leave the codebase in a better state than we found it._